README.md

```diff
@@ -4,3 +4,23 @@
 [![Stories in Ready](https://badge.waffle.io/MoodCat/MoodCat.me.svg?label=ready&title=Ready)](http://waffle.io/MoodCat/MoodCat.me)
 
 [![Throughput Graph](https://graphs.waffle.io/MoodCat/MoodCat.me/throughput.svg)](https://waffle.io/MoodCat/MoodCat.me/metrics)
+
+# Installation instructions
+
+To succesfully run/test the application, a couple of steps should be followed to install various dependencies and configurations:
+
+1. After cloning, open the IDE of your preference (Eclipse/IntelliJ), import the project as maven project and make sure `target/metamodel` is a resource-folder of your project.
+2. Execute the following Maven commands:
+  * `mvn generate-resources` This command should create various classes in `target/metamodel`
+  * `mvn install` This command should install all maven dependencies
+3. Install Lombok
+  * (Eclipse only) Close your IDE and run the following jar. `.m2/repository/org/projectlombok/lombok/LATEST_LOMBOK_VERSION/lombok-LATEST_LOMBOK_VERSION.jar`
+The installation will provide the steps to attach Lombok to your IDE.
+  * (IntelliJ only) Go to `preferences->plugins->browse repositories->lombok` and install the plugin.
+4. (Production only) Whenever you want to run the App in production, the database connection requires a password. Therefore it is not possible to launch if you don't add this password to the environment:
+    * (Eclipse only) Go to `run->run configurations->Java Application->App`.
+Click the `Environment` tab.
+    * (IntelliJ only) Go to `me.moodcat.core.App` and click on `edit configurations`.
+  * Click on `new`.
+As name set `database-password` and as value the database password.
```

Comment:
Does this use the correct persistence.properties? As it seems to me (after running) that it just runs the core with the inserted password.